bridgeservice/docs/swagger.json

```diff
@@ -436,6 +436,64 @@
                 }
             }
         },
+        "/remove-ger-events": {
+            "get": {
+                "description": "Returns a list of remove GER events, optionally filtered by block range or specific GER",
+                "produces": [
+                    "application/json"
+                ],
+                "tags": [
+                    "ger-events"
+                ],
+                "summary": "Get remove GER events",
+                "parameters": [
+                    {
```

Comment:
maybe we dont need these params
they can get all the removeger events all at once without any params
if they need some filtering, we can add